src/components/HoverCard/HoverCardOnUsername.tsx

```diff
@@ -0,0 +1,168 @@
+import { useState } from 'react';
+import colors from 'components/core/colors';
+import InteractiveLink from 'components/core/InteractiveLink/InteractiveLink';
+import { BaseM, TitleM } from 'components/core/Text/Text';
+import styled, { css, keyframes } from 'styled-components';
+import { graphql, useFragment } from 'react-relay';
+import { HoverCardOnUsernameFragment$key } from '__generated__/HoverCardOnUsernameFragment.graphql';
+import Markdown from 'components/core/Markdown/Markdown';
+import unescape from 'lodash/unescape';
+import FollowButton from 'components/Follow/FollowButton';
+import transitions, {
+  ANIMATED_COMPONENT_TRANSLATION_PIXELS_LARGE,
+} from 'components/core/transitions';
+import { HoverCardOnUsernameFollowFragment$key } from '__generated__/HoverCardOnUsernameFollowFragment.graphql';
+import { useLoggedInUserId } from 'hooks/useLoggedInUserId';
+
+type Props = {
+  username: string;
```

Comment:
could we get the username from the user fragment instead of passing it in as a prop?

Reply:
Definitely! Overlooked on this